docs/app/pages/components/components-sections/input-controls/number-picker/number-picker.component.html

```diff
@@ -1,14 +1,14 @@
 <p>Selecting an integer:</p>
 
-<ux-number-picker class="number-picker-component-demo" min="-10" max="10" [formControl]="form.controls['integer']"></ux-number-picker>
+<ux-number-picker class="number-picker-component-demo" min="-10" max="10" [formControl]="form.controls['integer']" [placeholder]="placeholder"></ux-number-picker>
```

Comment:
I think we should leave this out of the default example. If someone wants to test the placeholder input they can use the plunker to do so.

Reply:
Done the changes